src/debugger.ts

```diff
@@ -77,11 +77,10 @@ async function main(
     return config;
   }
 
+  // Sometimes VSCode launches then attaches, so having a warning/error here is confusing
+  // We used to return null in that case but that failed the attach.
```

Comment:
Does mirrord work in this case though? :thinking: 
If the service is already launched, we should not be injected

Reply:
We won't - we return config before doing any modifications.

Reply:
Just to be clear - what happens is that the process launches (with mirrord) then vscode does "attach" but that fails because we return null, now when we just return config the attach also works.

Reply:
Oh, ok!